src/modules/token/index.ts

```diff
@@ -0,0 +1,3 @@
+import * as tokenService from './token.service.ts';
```

Comment:
For me it doesn't look like we need a separated module called `token`, it's probably better to put it into `auth` module